util/paywall.go

```diff
@@ -28,8 +28,8 @@ import (
 const (
 	dcrdataMainnet = "https://explorer.dcrdata.org/api"
 	dcrdataTestnet = "https://testnet.dcrdata.org/api"
-	insightMainnet = "https://mainnet.decred.org/api"
-	insightTestnet = "https://testnet.decred.org/api"
+	insightMainnet = "https://mainnet.decred.org/insight/api"
+	insightTestnet = "https://testnet.decred.org/insight/api"
 
 	requestTimeout = 3 // Block explorer request timeout in seconds
```

Comment:
This isn't related to the changes, but I found it while testing this PR so we might as well fix it.  The request timeout is being passed in as nanoseconds when its suppose to be seconds. 

```suggestion
var (
	requestTimeout = 3 * time.Second // Block explorer request timeout
)
```